blindsign/blindrsa/blindrsa_test.go

```diff
@@ -0,0 +1,320 @@
+package blindrsa
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	_ "crypto/sha256"
+	"crypto/sha512"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"math/big"
+	"testing"
+)
+
+func TestSignVerify(t *testing.T) {
+	message := []byte("hello world")
+	key, err := rsa.GenerateKey(rand.Reader, 4096)
```

Comment:
`go test` takes about 10 seconds on my machine. This test alone takes 1.8 seconds. If you knock this down from 4096 to 2048 bits, it takes just 0.1 seconds. I applaud your conservative choice of RSA params, but maybe we should go for the minimum recommended for tests :)

Another way to speed up the tests would be to just generate a single key pair once and hard-code it in this file.

Here and below.

Reply:
Chatted offline. We'll replace this with a hard-coded key pair!

Reply:
Done.